viv-wallet-app/src/views/Wallets.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div class="wallets">
+        <loading v-bind:loading="loading" v-bind:errored="errored">
+            <section>
+                <h2>Wallets</h2>
+                <div class="buttons-container">
+                    <div class="selector-container">
+                        <label class="expertise-label" for="select-expertise">Expertise :</label>
+                        <div class="select">
+                            <select id="select-expertise" v-model="selectedExpertiseId" v-on:change="expertiseChanged">
+                                <option disabled value>Choisissez</option>
+                                <option
+                                    v-for="expertise in expertises"
+                                    :key="expertise.id"
+                                    v-bind:value="expertise.id"
+                                    >{{ expertise.expertiseName }}</option
+                                >
+                            </select>
+                            <span class="select-focus"></span>
+                        </div>
+                    </div>
+                </div>
+                <consultant-list v-bind:consultants="consultants" v-bind:expertise="selectedExpertiseId">
+                    <template v-slot="{ consultantId }">
+                        <router-link
+                            v-bind:to="`/wallets/${selectedExpertiseId}/${consultantId}`"
+                            class="tertiary-button update-button"
+                            tag="a"
+                            >Voir le wallet</router-link
+                        >
+                    </template>
+                </consultant-list>
+            </section>
+        </loading>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue, Watch } from "vue-property-decorator";
+import { Consultant } from "../models/consultant";
+import { Expertise } from "../models/expertise";
+import { ConsultantsService } from "@/services/consultants";
+import { ExpertisesService } from "@/services/expertises";
+import Loading from "../components/Loading.vue";
+
+import ConsultantList from "@/components/ConsultantList.vue";
+
+@Component({
+    name: "wallets",
+    components: { ConsultantList, Loading }
+})
+export default class Wallets extends Vue {
+    consultants: Consultant[] = [];
+    expertises: Expertise[] = [];
+    selectedExpertiseId = "";
+    loading = true;
+    errored = false;
+
+    expertisesService = new ExpertisesService();
+
+    async mounted() {
+        try {
+            this.expertises = await this.expertisesService.getExpertises();
+            if (this.$route.params.id) {
+                this.selectedExpertiseId = this.$route.params.id;
+                await this.updateConsultants();
+            } else {
+                if (this.expertises.length > 0) {
```

Comment:
there's something non trivial going on here...
why do you take the first one of a list with more than one value?

Reply:
ok... if there's no "selected" expertise, you force me to take one, right?

Reply:
Yes exactly. I used the same code as the consultants' page.